code/datums/uplink_item.dm

```diff
@@ -1063,6 +1063,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 8
 	excludefrom = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/suits/internals
+	name = "Syndicate Internals Box"
+	desc = "Standard issue syndicate internals. Contains an extended oxygen tank, a syndicate gas mask and an emergency auto-injector."
+	reference = "SIB"
+	item = /obj/item/storage/box/syndie_kit/internals
+	cost = 2
```

Comment:
This is uplink bloat.

Who would ever realistically buy this? Outside of super super niche edge cases that will happen once in a million rounds (I got spaced and someone took my survival box and I didn't happen to have a back up, and I happened to be a traitor and happened to not get my uplink stolen and happened to not get knocked out or cuffed in the process and I was able to activate my uplink and purchase it and it saved me and I got greentext because of it!"

Reply:
As I mentionned, traitor perma'd with uplink still available (implant/storage) is the main customer, prisoners don't get internal boxes.